.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,165 @@
+
+        > [!IMPORTANT]
+        > If you are using an older version, please update to the latest stable version and check if the issue persists before continuing this bug report.
+      options:
+        - "Please select a version."
+        - "- Unstable (25.05)"
+        - "- Stable (24.11)"
+        - "- Oldstable (24.05)"
+      default: 0
+    validations:
+      required: true
+  - type: "textarea"
+    id: "description"
+    attributes:
+      label: "Describe the Bug"
+      description: "Please include a clear and concise description of what the issue is."
+      placeholder: "Launching `hello` through the terminal on my ARM Linux machine results in the following error..."
+    validations:
+      required: true
+  - type: "textarea"
+    id: "how-to-reproduce"
+    attributes:
+      label: "Steps to Reproduce"
+      description: "Please include a step-by-step guide for reproducing this issue."
+      placeholder: |
+        1. Ensure you are running Nix on Linux in an ARM machine.
+        2. Launch `hello` using the terminal with `nix run nixpkgs#hello`.
+      value: |
+        1. ...
+        2. ...
+        3. ...
+    validations:
+      required: true
+  - type: "input"
+    id: "expected-behaviour"
+    attributes:
+      label: "Expected Behaviour"
+      description: "Please write a consise description of what was supposed to happen."
+      placeholder: "`hello` would launch normally."
+    validations:
+      required: true
+  - type: "textarea"
+    id: "screenshots"
+    attributes:
+      label: "Screenshots"
+      description: |
+        If applicable, add screenshots to help explain your problem.
+        If you need help uploading images to GitHub, please review the [relevant documentation](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#uploading-assets).
+    validations:
+      required: false
+  - type: "textarea"
+    id: "logs"
+    attributes:
+      label: "Relevant log output"
+      description: |
+        If applicable, copy and paste any relevant log output.
+        This will be automatically formatted into code, so no need for backticks.
+      render: "console"
+    validations:
+      required: false
+  - type: "textarea"
+    id: "additional-context"
+    attributes:
+      label: "Additional Context"
+      description: "Add any other context about the problem here."
+      placeholder: "This issue only happens when my desktop environment is running on X11."
+    validations:
+      required: false
+  - type: "textarea"
+    id: "metadata"
+    attributes:
+      label: "System Metadata"
+      description: "Please run `nix-shell -p nix-info --run \"nix-info -m\"` on a terminal and paste the output of that command here."
+      placeholder: |
+        - system: `"aarch64-linux"`
+        - host os: `Linux 1.0.0, NixOS, 17.09 (Hummingbird), a31ce27`
```

Comment:
We could probably use a more modern example... 😅

Reply:
Marking as resolved as all placeholders were removed.